app/models/thredded/user_topic_follow.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module Thredded
+  class UserTopicFollow < ActiveRecord::Base
+    REASON_MANUAL = 'manual'
+    REASON_POSTED = 'posted'
+    REASON_MENTIONED = 'mentioned'
```

Comment:
You can use symbols when assigning. When checking, you can use e.g. `user_topic_follow.mentioned?`.